### ISSUE TITLE
Request - mehrere Fenster automatisch anordnen

### ISSUE THREAD

**Issue by aisbergde** [Thomas-Mielke-Software/EasyCash#10]
Manchmal will man die Daten verschiedener Jahre nebeneinander angezeigt bekommen. Da wäre doch ein automatisches Anordnen der Fenster, wie das viele Windows Programme haben, nicht schlecht. Dan braucht man das nicht immer manuell anzupassen.

**Comment by thomiel** [comment]
Ja, das wäre moderner. Ein 'Fenster'-Knopf im Ansicht-Menübereich, ähnlich dem Journal-Ansicht-Knopf vielleicht...

**Comment by aisbergde** [comment]
Früher hatten viele Programme einen Punkt "Windows" im Menü, und da konnte man dann die Anordnung auf Knopfdruck ändern. Heutzutage arbeiten immer mehr mit Tabs, floating und angedockten Fenstern usw. Alles schön und toll. Doch manchmal wünsche ich mir einfach zusätzlich diese alte Funktionalität zurück. Das ist freilich nur dann sinnvoll, wenn es vom verwendeten Toolkit zum Erstellen einer GUI "out-of-the-box" unterstützt wird.

Ich habe mal getestet, ob ich einfach zwei Instanzen des Programms öffne und diese dann einfach mit wenigen Tasten-Klicks nebeneinander anordne. Das scheint grundsätzlich zu funktionieren und erleichtert auch das Umherspringen. Ich hoffe, das ist so vorgesehen und die beiden Instanzen stören sich nicht gegenseitig?

**Comment by thomiel** [comment]
Nein, solange man nicht versucht in beiden Instanzen Einstellungen zu ändern oder das gleiche Jahr zu bearbeiten.
